2001-3000.po

```diff
@@ -26,37 +26,37 @@ msgstr "Łóżko Medyczne"
 #. STRINGS.BUILDINGS.PREFABS.MESHTILE.DESC
 msgctxt "STRINGS.BUILDINGS.PREFABS.MESHTILE.DESC"
 msgid "How do Duplicants keep their tiny feet from slipping through the holes?"
-msgstr ""
+msgstr "Jak Duplikanci mogą utrzymywać swoje malutkie stópki na takiej powierzchni?"
 
 #. STRINGS.BUILDINGS.PREFABS.MESHTILE.EFFECT
 msgctxt "STRINGS.BUILDINGS.PREFABS.MESHTILE.EFFECT"
 msgid "Can be used as wall or floor tile to enclose rooms.\n\nDoes not obstruct the flow of <style=\"liquid\">Liquid</style> or <style=\"gas\">Gas</style>."
-msgstr ""
+msgstr "Ta płytka może być zastosowana jako ściana lub podłoga.\n\nNie blokuje przepływu <style=\"liquid\">Cieczy</style> i <style=\"gas\">Gazu</style>."
 
 #. STRINGS.BUILDINGS.PREFABS.MESHTILE.NAME
 msgctxt "STRINGS.BUILDINGS.PREFABS.MESHTILE.NAME"
 msgid "Mesh Tile"
-msgstr "Płyta Siatki"
+msgstr "Kratka"
 
 #. STRINGS.BUILDINGS.PREFABS.MICROBEMUSHER.DESC
 msgctxt "STRINGS.BUILDINGS.PREFABS.MICROBEMUSHER.DESC"
 msgid "Apparently Duplicants can glean some nutrition from the various goops this outputs."
-msgstr ""
+msgstr "Najwyraźniej Duplikanci korzystając z różnych lepkich substancji są w stanie zdobyć trochę pożywienia."
 
 #. STRINGS.BUILDINGS.PREFABS.MICROBEMUSHER.EFFECT
 msgctxt "STRINGS.BUILDINGS.PREFABS.MICROBEMUSHER.EFFECT"
 msgid "Produces low quality <style=\"food\">Food</style> for Duplicants using common ingredients.\n\nDuplicants will not fabricate unless recipes are queued."
-msgstr ""
+msgstr "Produkuje <style=\"food\">Jedzenie</style> niskiej jakości, z użyciem pospolitych składników.\n\nDuplikancji nie będą wykonywać dań, dopóki nie zostanie utworzona kolejka dań."
 
 #. STRINGS.BUILDINGS.PREFABS.MICROBEMUSHER.NAME
 msgctxt "STRINGS.BUILDINGS.PREFABS.MICROBEMUSHER.NAME"
 msgid "Microbe Musher"
-msgstr "Konwenter Żywności"
+msgstr "Zgniatarka drobnoustrojów"

```

Comment:
Powiem szczerze, że "Konwerter Żywności" brzmi mi lepiej. Bo od razu sugeruje, do czego to służy. A idealnie oddać znaczenia się nie da ;)